### PR TITLE
:sparkles: validate integration with antropy

### LINKED ISSUE #85: Check integration compatibility with Antropy
- [ ] add this to the testing matrix

### PR DESCRIPTION
**Pull request by jvdd**
To integrate with `antropy` we require no wrappers!! :tada: 
On the following 2 functions should be decorated with a `FuncWrapper`:
- `spectral_entropy` -> requires mandatory `sf` argument
- `hjorth_params` -> has 2 outputs (thus needs to add these as `output_names`)

Resolves #85 

- [x] add test for integration with `antropy`
  - [x] add test for `float64` dtype
- [x] add example notebook

### CODE REVIEW

File: tests/test_features_integration.py [hunk 1]
```diff
@@ -322,3 +322,41 @@ def test_catch22_all_features(dummy_data):
...
+
+## ANTROPY
+
+
+def test_antropy_all_features(dummy_data):
+    # Tests if we integrate with ALL antropy features
+    # -> this requires no additional wrapper!
+    import antropy as ant
+
+    funcwrapper_entropy_funcs = [
```
Comment (jonasvdd) [inline tests/test_features_integration.py:335]: This is already rather niche; when we need a func-wrapper; as such I would include this somewhere in an example noteobok

Comment (jvdd) [reply tests/test_features_integration.py:335]: Sure! However, we need a funcwrapper quite often.

Guess we should make an example notebook for the following:
- pycatch22
- antropy

Comment (jonasvdd) [reply tests/test_features_integration.py:335]: indeed!

Comment (jvdd) [reply tests/test_features_integration.py:335]: Hmm as only a funcwrapper is necessary for antropy, I would rather just add this to the readme in examples than creating a full-blown notebook for it?

**Comment by jvdd** [comment]
When further looking into this, I observed the numba code in `antropy` throws an error when the data is a "non-writable" array.

We could solve this by creating a new PR to `antropy` that applies the fix of this PR https://github.com/raphaelvallat/antropy/pull/13 to the other numba accelerated feature functions.

**Comment by jvdd** [comment]
> When further looking into this, I observed the numba code in `antropy` throws an error when the data is a "non-writable" array.
> 
> We could solve this by creating a new PR to `antropy` that applies the fix of this PR [raphaelvallat/antropy#13](https://github.com/raphaelvallat/antropy/pull/13) to the other numba accelerated feature functions.

Looked into the antropy code base and submitted a fix for this issue https://github.com/raphaelvallat/antropy/pull/23

**Comment by jonasvdd** [comment]
@jvdd, could you elaborate what you imply with the `test float64 dtype`?

**Comment by jvdd** [comment]
> @jvdd, could you elaborate what you imply with the `test float64 dtype`?

See antropy PR that I linked above

**Comment by jvdd** [comment]
> @jvdd, could you elaborate what you imply with the `test float64 dtype`?

Added this to the feature integration tests! Using the Antropy github as dev dependencies now makes the test pass - as soon as Antropy has a new release we can substitute this!

File: tests/test_features_integration.py (hunk 1 above)
Comment (jvdd) [reply tests/test_features_integration.py:335]: Eventually ended up making an example notebook :upside_down_face:

**Comment by jvdd** [comment]
Eventually removed the antropy example notebook. Ready for review now @jonasvdd :rocket:

**Review by jonasvdd — APPROVED** [review]
LGTM!

**Pull request merged**